spec/integrations/quickbooks_online/deposits/update_spec.rb

```diff
@@ -9,34 +11,32 @@
   include AdaptorHelpers
   include QuickBooksHelpers
 
-  before {
+  before do
     stub_find_deposit
     stub_update_deposit
-  }
+  end
 
   let(:account) do
-    LedgerSync::Account.new(account_resource({ledger_id: '123'}))
+    LedgerSync::Account.new(account_resource(ledger_id: '123'))
   end
 
   let(:line_item_1) do
-    LedgerSync::DepositLineItem.new(deposit_line_item_resource({account: account}))
+    LedgerSync::DepositLineItem.new(deposit_line_item_resource(account: account))
```

Comment:
I was using `{}` to make sure its clear were working with a hash and not keyword arguments.